spec/models/book_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe Book do
+  subject(:book) { create(:book) }
+
+  it '#validate' do
+    is_expected.to validate_presence_of :genre
+    is_expected.to validate_presence_of :author
+    is_expected.to validate_presence_of :image
+    is_expected.to validate_presence_of :title
+    is_expected.to validate_presence_of :publisher
+    is_expected.to validate_presence_of :year
```

Comment:
We usually have each of this validations inside a particular it, so it is easier to identify which one fails later. Also, we use parenthesis when using params.
`it { is_expected.to validate_presence_of(:genre) }`
`it { is_expected.to validate_presence_of(:author) }` and so on

Reply:
Ok